legacy_promotions/db/migrate/20160101010001_solidus_one_four_promotions.rb

```diff
@@ -122,5 +122,17 @@ def up
         t.index ["starts_at"], name: "index_spree_promotions_on_starts_at"
       end
     end
+
+    unless table_exists?(:spree_line_item_actions)
```

Comment:
I think we need a new migration file now that we already shipped this gem 🙈

Reply:
I think we actually don't. This is just moved from the rollup migration for solidus one four in core to the one in solidus_legacy_promotions, and people who have installed Solidus have the table.

Reply:
Right